mcode/mcode-coverage-checker
# Adding utility for checking coverage of patient resources

# Description
Issue: STEAM-841

This PR adds coverage checking for Patient resources, which checks if the fields under patient in the mCODE diagram are present in the resources.

## Important Changes
New file `patientCoverage.js` that contains the function `getPatientCoverage()` which takes in a bundle and returns an array of objects, where the objects have the patient fields as written in the mCODE diagrams as keys and boolean values indicating if the field is covered or not.

## Testing Recommendations
For testing purposes, I have added the file `bundle.json` which is the Jenny M example again, but with an added empty patient resource. There is also a `console.log()` statement for testing purposes. I'll take these out before merging.
- Make sure that each fields has a proper true/false value when it is covered or not
- Make sure the inputs and outputs of the function make sense within the larger app

# Checklists

**Submitter:**

@ACCT1 :
